app/views/works/_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<% page_title ||= "Add / Update" %>
+<h3><%= page_title %></h3>
+
+
+<section>
+  <%= form_for @work do |f| %>
+
+    <%= f.label :category %>
+    <% categories = ['album', 'book', 'movie'] %>
```

Comment:
These might make sense as constants in the model rather than in the view - though I'm not sure if you're using them in any other places